src/services/thermalOvercrowdingService.ts

```
@@ -11,6 +11,45 @@ export interface ThermostatReading {
   venue_id: string;
   temperature_fahrenheit: number;
   recorded_at: string;
+  humidity_percent?: number;
+  occupancy_count?: number;
+}
+
+export type MentalHealthThermalRiskLevel =
+  | 'NORMAL'
+  | 'MODERATE'
+  | 'ELEVATED'
+  | 'CRITICAL_SENSORY_OVERLOAD';
```

ESLint

[error] Replace `|·'NORMAL'⏎··|·'MODERATE'⏎··|·'ELEVATED'⏎··|·'CRITICAL_SENSORY_OVERLOAD'` with `"NORMAL"·|·"MODERATE"·|·"ELEVATED"·|·"CRITICAL_SENSORY_OVERLOAD"`

(prettier/prettier)